packages/docs/src/pages/guides/styled-components.mdx

```diff
@@ -32,16 +37,20 @@ import styled from 'styled-components'
 import styled from '@emotion/styled'
 ```
 
+[@emotion/styled]: https://emotion.sh/docs/styled
+[styled components]: https://styled-components.com
+
 ## Using the `sx` prop
 
-To avoid the need for an additional dependency,
-you can use the `sx` prop to create custom styled components.
+Theme UI is actively working on removing its internal dependency on `@emotion/styled` to reduce bundle size.
```

Comment:
> Theme UI is actively working on removing its internal dependency on `@emotion/styled` to reduce bundle size.

I'm not sure if this information is relevant here.

Reply:
I debated, but I want people who set them up now to know there's a breaking change coming in the next few months